main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/IdeStartup.cs

```diff
@@ -209,6 +209,8 @@ int Run (MonoDevelopOptions options)
 
 			IdeApp.IsRunning = true;
 
+			var commandService = Runtime.GetService<CommandManager> ().Result;
```

Comment:
Why do we need to load it here?

Reply:
Maybe add a doc comment? and use discard?

```suggestion
			_ = Runtime.GetService<CommandManager> ().Result;
```